VAMobile/src/screens/HealthScreen/SecureMessaging/ReplyHelp/ReplyHelp.tsx

```diff
@@ -56,13 +56,7 @@ const ReplyHelp: FC = () => {
             ]}
           />
         </Box>
-
-        <ClickForActionLink
-          displayedText={t('secureMessaging.replyHelp.call911')}
-          a11yLabel={t('secureMessaging.replyHelp.call911.a11y')}
-          numberOrUrlLink={t('911')}
-          linkType={LinkTypeOptionsConstants.call}
-        />
+        <ClickToCallPhoneNumber ttyBypass phone={t('911')} a11yLabel={t('secureMessaging.replyHelp.call911.a11y')} displayedText={t('secureMessaging.replyHelp.call911')} />
```

Comment:
I don't love the shorthand of 'ttyBypass' here...feels like syntactical sugar that can be confusing to me. Pretty sure this'd be the only place in the entire app we do this, so based on just a consistency metric I think we should change it to the suggested.
```suggestion
        <ClickToCallPhoneNumber ttyBypass={true} phone={t('911')} a11yLabel={t('secureMessaging.replyHelp.call911.a11y')} displayedText={t('secureMessaging.replyHelp.call911')} />
```

Reply:
Gotchya. Done. (I did find one other place, added it there, too.)